app/models/company.rb

```diff
@@ -47,4 +48,8 @@ def create_new_user
   def email_new_user(user, password)
     UserEmailer.send_welcome_email(user, password).deliver_now
   end
+
+  def verify_hire_count
+    hire_count = 0 if hire_count == ""
+  end
```

Comment:
Hoping this would of solved the issue, and I feel like it's close! We'll have to go back and take a look at this.